docs/clipboard-unsanitized/HOWTO.md

```diff
@@ -0,0 +1,40 @@
+# How to use the Async Clipboard API to read and write unsanitized HTML
+**Last updated: March, 2023**
+
+Reading and writing unsanitized HTML to and from the clipboard is currently available in Chromium-based browsers in 113 and later behind the flag `ClipboardUnsanitizedContent`.
+1. Download Microsoft Edge ([Canary Channel](https://www.microsoftedgeinsider.com/en-us/download/canary)).
+2. Launch Edge with the command line flag `--enable-blink-features=ClipboardUnsanitizedContentNavigate`.
+
+## Example
+
+The write method doesn't change it's shape:
```

Comment:
Nit: Typo: The write method doesn't change **its** shape

Reply:
Also would it be better to say something like " There is no change in the API shape for write() method"?

Reply:
Done, thanks for the suggestion.